src/components/Skeleton/Skeleton.tsx

```
@@ -0,0 +1,360 @@
+// 차트 스켈레톤
+export const SkeletonChart: React.FC<{
+  height?: string;
+  className?: string;
+}> = ({ height = "h-64", className = "" }) => (
+  <div className={`${height} ${className}`}>
+    <div className="h-full flex items-end gap-2 px-4 pb-4">
+      {Array.from({ length: 8 }).map((_, i) => (
+        <div key={i} className="flex-1 flex flex-col justify-end">
+          <Skeleton
+            className="w-full rounded-t"
+            style={{ height: `${30 + Math.random() * 50}%` }}
```

ESLint

[error] Error: Cannot call impure function during render

`Math.random` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/components/Skeleton/Skeleton.tsx:56:38
  54 |           <Skeleton
  55 |             className="w-full rounded-t"
> 56 |             style={{ height: `${30 + Math.random() * 50}%` }}
     |                                      ^^^^^^^^^^^^^ Cannot call impure function
  57 |           />
  58 |         </div>
  59 |       ))}

(react-hooks/purity)